docs/framework-integration/vue.md

```diff
@@ -281,24 +319,198 @@ createApp(App).use(ComponentLibrary).mount('#app');
 
 In your page or component, you can now import and use your component wrappers:
 
-```ts
+```html
 <template>
```

Comment:
For my own edification, why do we need to use `<template>` here?

Reply:
Because we are using [Single-file components (SFC)](https://vuejs.org/guide/scaling-up/sfc.html)